controllers/category.controller.js

```diff
@@ -0,0 +1,96 @@
+const { extend } = require("lodash");
+const mongoose = require("mongoose");
+const { Category } = require("../models/Category.model");
+const { Video } = require("../models/Video.model");
+const getAllCategories = async (req, res) => {
+  try {
+    const allCategories = await Category.find({});
+    res.status(200).json({
+      success: true,
+      message: "All Categories",
+      categories: allCategories,
+    });
+  } catch (err) {
+    res.status(400).json({
+      success: false,
+      message: "Error in getting categories",
+      errMessage: err.errMessage,
+    });
+  }
+};
+const addACategory = async (req, res) => {
+  const categoryDetails = req.body;
+  const newCategory = new Category(categoryDetails);
+  let savedCategory;
```

Comment:
Declare it as const you are not changing its value here, right? Then change from let to const